lib/js_browser.mli

```diff
@@ -17,6 +17,61 @@ module Storage : sig
   val clear: t -> unit
 end
 
+module Date : sig
+  type t = private Ojs.t
+  val t_of_js: Ojs.t -> t
+  val t_to_js: t -> Ojs.t
+
+  val new_date: int -> t [@@js.new]
+  val now : unit -> int [@@js.global "Date.now"]
```

Comment:
OCaml ints are represented using the Number class in JS, with a 32-bit signed semantics.  I think the faithful representation of JS Number would rather be OCaml's float type.  You cannot represent the number of millisecond since 1970 using a 32-bit integer (even at second-level, this would overflow in 2032).